experimental/PropertyDDS/packages/property-shared-tree-interop/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { convertPSetSchemaToSharedTreeLls as convertPSetSchema } from "./schemaConverter";
```

Comment:
Is `convertPSetSchema` a good public name for the feature?

Reply:
Fixed in 09d3220fdbf342dd733b0a0c30e4eda579ff3a49